contracts/0.4.24/Lido.sol

```diff
@@ -1096,19 +909,38 @@ contract Lido is Versioned, StETHPermit, AragonApp {
         uint256 clValidators = CL_VALIDATORS_POSITION.getStorageUint256();
         // clValidators can never be less than deposited ones.
         assert(depositedValidators >= clValidators);
+
         return (depositedValidators - clValidators).mul(DEPOSIT_SIZE);
     }
 
     /**
-     * @dev Gets the total amount of Ether controlled by the system
+     * @dev Gets the maximum allowed external balance as basis points of total pooled ether
+     * @return max external balance in wei
+     */
+    function _getMaxExternalEther() internal view returns (uint256) {
+        return _getPooledEther()
+            .mul(MAX_EXTERNAL_BALANCE_POSITION.getStorageUint256())
+            .div(TOTAL_BASIS_POINTS);
+    }
+
+    /**
+     * @dev Gets the total amount of Ether controlled by the protocol
      * @return total balance in wei
      */
-    function _getTotalPooledEther() internal view returns (uint256) {
+    function _getPooledEther() internal view returns (uint256) {
```

Comment:
Not really intuitive thing. Why there is "pooled" and "total pooled"? What is the difference?